elyra/pipeline/tests/test_component_parser_airflow.py

```diff
@@ -58,7 +58,7 @@ def test_parse_airflow_component_file():
 
     # Ensure component parameters are prefixed (and system parameters are not), and hold correct values
     assert properties_json['current_parameters']['label'] == ''
-    assert properties_json['current_parameters']['component_source'] == entry.location
+    assert properties_json['current_parameters']['component_source'] == entry['location']
```

Comment:
I think this meant to be `component_entry.location` or scrap the simple namespace thing altogether.

Reply:
Ah completely missed this! Yeah I should have accessed `component_entry.location`, not sure why I didn't